### PR TITLE
Feature api vis3 data reformat

### PR DESCRIPTION
**Pull request by NortonDJ**
Fix the pull of the data.

Axes will need to be fixed, but @cencije should handle that

### CODE REVIEW

File: src/pages/Visualization3.jsx [hunk 1]
```diff
@@ -8,19 +8,24 @@ export default class Visualization3 extends Component {
...
+      data: null
     }
   }
 
   componentDidMount() {
     fetch('http://localhost:3001/api/v1/visualizations/3').then((res) => {
       return res.json()
     }).then((data) => {
-      this.setState({ data })
+      // console.log("Data Received: " + JSON.stringify(data))
```
Comment (wassgha) [inline src/pages/Visualization3.jsx:19]: Remove

File: src/api/app/controllers/api_controller.rb [hunk 2]
```diff
@@ -68,18 +68,32 @@ def visualization_2
...
               .where.not(data_school_facts: { sat_writing: nil })
-              .order('revenue ASC')
+              .order('totalRevenue ASC')
               .all()
-    json_response(@fiscal_information)
+
+    json_response(visualization_3_custom_json(@fiscal_information))
+  end
+
+  def visualization_3_custom_json(data)
```
Comment (wassgha) [inline src/api/app/controllers/api_controller.rb:86]: Remove, give raw json from the database and change format on the UI side instead

File: src/api/app/controllers/api_controller.rb [hunk 3]
```diff
@@ -68,18 +68,32 @@ def visualization_2
   def visualization_3
     @fiscal_information = School.joins(:Fiscal, :Fact)
               .select('
-                (local_revenue + state_revenue + other_revenue + fed_revenue) AS revenue,
-                school.state_lea_id,
```
Comment (wassgha) [inline src/api/app/controllers/api_controller.rb:72]: Why was this deleted? Could be useful for svg keys

**Review by wassgha — CHANGES_REQUESTED** [review]

File: src/api/app/controllers/api_controller.rb (hunk 3 above)
Comment (NortonDJ) [reply src/api/app/controllers/api_controller.rb:72]: Unless Joe does some filtering on the data, you're giving him the wrong format of the data which breaks the visualization. I'm giving him exactly what he needs for his visualization. Having the state_lea_id in the data would cause issues. Maybe there is another way around this, but I think it should be the case that we give him what he asks for exactly according to data_03.js file.

**Review by wassgha — APPROVED** [review]
